module/proto/gravitynft/v1/types.proto

```diff
@@ -13,9 +13,28 @@ message LastObservedNFTEthereumBlockHeight {
     uint64 ethereum_block_height = 2;
 }
 
-// TODO: Do we need UnhaltBridgeProposal?
```

Comment:
I think we do. NFT bridge should be separately stoppable from erc20 one, right?

Reply:
Yep, makes sense